nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -1548,4 +1552,20 @@ struct nrf_wifi_event_deinit_done {
 	struct nrf_wifi_sys_head sys_head;
 } __NRF_WIFI_PKD;
 
+/**
+ * @brief This structure represents the event that provides UMAC (Upper MAC) internal
+ *  memory statistics in response to the NRF_WIFI_CMD_GET_DEBUG_STATS command.
+ *
+ */
+struct nrf_wifi_umac_event_debug_stats {
+	/** UMAC header, @ref nrf_wifi_sys_head */
+	struct nrf_wifi_sys_head sys_head;
+	/** rpu_lockup_event count */
```

Comment:
Can you please add more details that explains the meaning of this statistics rather than duplicating the name?

Reply:
Updated